provisioning/service/src/ProvisioningServiceClientFactory.cs

```diff
@@ -36,19 +34,21 @@ public static ProvisioningServiceClient CreateFromConnectionString(String connec
         /// <returns></returns>
         public static ProvisioningServiceClient CreateFromConnectionString(String connectionString, HttpClientHandler rootHandler, params DelegatingHandler[] handlers)
```

Comment:
Per our last meeting, please change this into an extension method (add `this ProvisioningServiceClient` as the first parameter).